lib/sparql_to_sw_solr/instance_title_fields.rb

```diff
@@ -34,8 +34,7 @@ def primary_title_result
               <#{instance_uri}> bf:responsibilityStatement ?o .
               ?ignore ?p ?o
             }
-          }".freeze
-        sparql.query(query)
+          }"
```

Comment:
This could be a misunderstanding of the code - it returns results.  We can punt on all of this until later - moving this refactoring to #34